### ISSUE TITLE
Does Clipper support multiple workers?

### ISSUE THREAD

**Issue by aksnzhy** [ucbrise/clipper#520]
I have 4 GPUs on my PC and I want to use them to serve one model in a load-balanced way. Does Clipper support that?

**Comment by dcrankshaw** [comment]
Yes it does. You can create [multiple replicas of a model](http://docs.clipper.ai/en/v0.3.0/clipper_connection.html#clipper_admin.ClipperConnection.set_num_replicas) and Clipper will automatically load-balance requests between them. However, Clipper does not yet support deploying models on GPUs automatically. You can instead do this manually by starting the Docker containers yourself using `nvidia-docker` after registering your model. You can look at the [function](https://github.com/ucbrise/clipper/blob/develop/clipper_admin/clipper_admin/docker/docker_container_manager.py#L207-L238) that Clipper calls to start containers to see which labels and environment variables are normally set.

**Comment by lgendrot** [comment]
I may be wrong, but instead of starting the docker containers yourself manually, you can simply [make the nvidia-docker runtime the default docker runtime](https://github.com/NVIDIA/nvidia-docker/wiki/Advanced-topics#default-runtime)

I haven't tested it, but presumably this would make every docker container started on your system have GPU access.
